server.js

```diff
@@ -65,7 +70,7 @@ app.post('/getParticularStudent', (req, res, next) => {
   // }
   db.loadParticular('student', req.body.studentInfo)
     .then((data) => {
-      res.end('Here is the requested student: \n' + JSON.stringify(data, undefined, 2));
+      res.end(JSON.stringify(data, undefined, 2));
```

Comment:
Good change, I meant to do that, but I guess I forgot.
Also, technically it should just be JSON.stringify(data)not JSON.stringify(data, undefined, 2), because all the other 2 arguments do is add extra white space. While it is nice for use as human readers (particularly while testing), all it does for the computer is make the file a little bit bigger. Not an actual issue for our project, but a good practice in general

Reply:
Good point, I can make that change with the next PR.